saleor/checkout/calculations.py

```diff
@@ -490,30 +490,26 @@ def _get_checkout_base_prices(
         unit_price = base_calculations.apply_checkout_discount_on_checkout_line(
             checkout_info, lines, line_info, discounts, unit_price
         )
-        line_total_price = quantize_price(unit_price * quantity, unit_price.currency)
+        line_total_price = quantize_price(unit_price, unit_price.currency) * quantity
```

Comment:
I can't speak about the rest of this PR, but this particular change seems incorrect. While Saleor does not currently support fractional quantities, if we did add support for them, a quantity of `0.01` would result in USD line totals with four decimal places. Line totals need to be quantized after multiplication, so if adding quantization to unit prices helps, it's probably indicating a bug in another area in our logic.

Reply:
Revert this one.